### ISSUE TITLE
common metadata about KS components

### ISSUE THREAD

**Issue by tgbugs** [NeuroscienceKnowledgeSpace/KnowledgeSpace#18]
I think we need to come up with a common template for data sources, ontology sources, literature sources, etc. That way we can talk about some common metadata for all them. For example we want to be able to talk about the website, git repo, and how to contribute/contribution model for the ontology and for the data space in general, but also for specific datasets (e.g. alerting users that they could contribute data to a source like openfmri directly but not to allen cell types).

Short version: we need tables that appear on pages like
http://kaa.neuinfo.org/NeuroKS/index.php/Contributors and
http://kaa.neuinfo.org/NeuroKS/index.php/Documentation
that have |component|link|contribution model|etc.

**Comment by tgbugs** [comment]
Mathew has done the work for this. Need to find it and integrate it into the scr registry.

**Comment by tgbugs** [comment]
Additional issues.  
https://github.com/OpenKnowledgeSpace/KnowledgeSpace/issues/28
